raiden/lightclient/handlers/light_client_message_handler.py

```diff
@@ -42,6 +42,16 @@ def build_light_client_protocol_message(identifier: int, message: Message, signe
 class LightClientMessageHandler:
     log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
+    @classmethod
+    def is_message_already_stored(cls,
```

Comment:
Why is this method needed? also.. this change is added on the settlement light pr,can u explain why?

Reply:
that's what we talked about in slack